cargo-shuttle/src/lib.rs

```diff
@@ -343,16 +345,42 @@ impl Shuttle {
         Ok(())
     }
 
-    async fn logs(&self, client: &Client, id: Option<Uuid>, follow: bool) -> Result<()> {
+    async fn logs(
+        &self,
+        client: &Client,
+        id: Option<Uuid>,
+        latest: bool,
+        follow: bool,
+    ) -> Result<()> {
         let id = if let Some(id) = id {
             id
         } else {
-            let summary = client.get_service(self.ctx.project_name()).await?;
+            let proj_name = self.ctx.project_name();
+            let summary = client.get_service(proj_name).await?;
 
             if let Some(deployment) = summary.deployment {
+                // Active deployment
                 deployment.id
+            } else if latest {
```

Comment:
Should we perhaps always get the latest if the latest flag is passed in? So with no flag we try to get the current running one, with the flag we try to get the latest one. Because a running deployment might not always be the last deployment.

Reply:
Sounds good